addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1963,7 +1964,16 @@ export class Wysiwyg extends Component {
                 this.$editable.find(containerSelector)[0];
             if (container) {
                 const range = document.createRange();
-                range.selectNodeContents(container);
+                if (!container.hasChildNodes()) {
+                    range.selectNodeContents(container);
+                } else {
+                    let start = container.firstChild;
+                    while((!isVisible(start)) && start.nextSibling) {
+                        start = start.nextSibling;
+                    }
+                    range.setStart(start, 0);
+                    range.setEnd(...endPos(container.lastChild));
```

Comment:
What you needed to do to find the start node, how come you don't need to do it to find the end node as well?